src/Cassandra.IntegrationTests/Mapping/Tests/Fetch.cs

```diff
@@ -462,6 +462,27 @@ public void Fetch_TimeUuid_HashSet_Key()
         }
 
         [Test]
+        public void Fetch_TimeUuid_List()
+        {
+            _session.Execute("CREATE TABLE tbl_timeuuid_hashset (id int primary key, my_set set<timeuuid>)");
+            var map = new Map<PocoWithCollections<TimeUuid>>()
+                .Column(p => p.Id)
+                .Column(p => p.List, c => c.WithName("my_set"))
```

Comment:
Mapping a set to a `List` would be allowed but it would cause sorting issues, like the failures in the build: https://ci.appveyor.com/project/DataStax/csharp-driver/build/CSHARP-511-671